packages/netlify-cms-core/src/reducers/entries.ts

```diff
@@ -144,8 +144,7 @@ export const selectMediaFolder = (
 ) => {
   let mediaFolder = config.get('media_folder');
 
-  const useWorkflow = config.get('publish_mode') === EDITORIAL_WORKFLOW;
```

Comment:
All backends support committing multiple files in one commit now, so no need to limit bundling to editorial workflow only (this was used to limit the feature to the GitHub/Test backends)

Reply:
Awesome